src/main/java/com/auth0/json/mgmt/organizations/Invitation.java

```diff
@@ -44,14 +51,16 @@
     @JsonProperty("organization_id")
     private String organizationId;
     @JsonProperty("roles")
-    private Roles roles;
+    @JsonDeserialize(using = StringArrayDeserializer.class)
+    @JsonSerialize(using = StringArraySerializer.class)
+    private String[] roles;
```

Comment:
Instead of using a `String[]`, can't we just use a `List<String>`, and keep the `@JsonIgnore` on the getter/setter?

Reply:
That actually works, thanks! I fixed that in https://github.com/auth0/auth0-java/pull/380/commits/d682b2f2a1ff615a69f3001aabdb057cf8ddd7f9